src/main/java/build/buildfarm/worker/ReportResultStage.java

```diff
@@ -201,10 +201,8 @@ protected OperationContext tick(OperationContext operationContext) {
 
     try {
       worker.instance.putAllBlobs(contents.build());
-    } catch (IOException ex) {
-    } catch (InterruptedException ex) {
-      poller.stop();
-      return null;
+    } catch (IOException e) {
+      e.printStackTrace();
```

Comment:
Doesn't IOException here (in theory) mean that you failed to report results, and the action should be an INTERNAL_ERROR?

Reply:
Figured you'd catch that, but I wonder how much sense it really makes to report that as an error on the operation (I'm on board with an early exit, though). The IOException is actually a channel for RetryException, which means that we exhausted all possible attempts to communicate with the CAS, which was most likely transiently unavailable, so we should now put the operation into an error state (by again communicating to the same service that we just failed to) rather than letting some other worker that didn't have a beaver gnawing through half of its uplinks try it?

Reply:
Right, sorry, I misread the code. Agreed that it makes most sense to let some other worker try this, but this is better achieved by poller.stop; return null; here rather than continuing. Because if these were transient beavers, then putOperation might succeed and the client will get a successful response, causing their build to fail (because Bazel will not retry if it fails to download action results).

Reply:
These are low voltage lines, the beavers are much more likely to be nomadic than transient.

(switched to `poller.stop(); return null;`)